core/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>puppetlabs</groupId>
+    <groupId>puppetlabs.org.jruby</groupId>
     <artifactId>jruby-parent</artifactId>
     <version>1.7.26-puppetlabs-1-SNAPSHOT</version>
```

Comment:
Could go either way, but to me it probably makes more sense to version it relative to the version we think their changes will be shipping in.  I could change the final `-1` to `-rc1` or something, if that seems better.